pkgs/games/anki/addons/puppy-reinforcement/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "puppy-reinforcement";
+  version = "1.1.1";
+  src = fetchFromGitHub {
+    owner = "glutanimate";
+    repo = "puppy-reinforcement";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-y52AjmYrFTcTwd4QAcJzK5R9wwxUSlvnN3C2O/r5cHk=";
+  };
+  sourceRoot = "${finalAttrs.src.name}/src/puppy_reinforcement";
+  passthru.updateScript = nix-update-script { };
+  meta = {
+    description = "Anki add-on that encourages learners with pictures of cute puppies";
+    longDescription = ''
+      Puppy reinforcement is an Anki add-on that displays
+      images in variable intervals while learning.
+
+      The default puppy images can be replaced with custom ones at the user_files directory
+
+      The config options for withConfig can be found at
+      https://github.com/glutanimate/puppy-reinforcement/blob/v${finalAttrs.version}/src/puppy_reinforcement/config.md
+    '';
```

```suggestion
    longDescription = ''
      The options to configure this add-on can be found [here](https://github.com/glutanimate/puppy-reinforcement/blob/v${finalAttrs.version}/src/puppy_reinforcement/config.md).
      Extra images can also be added with `userFiles`.
      
      Example:
      
      ```nix
      pkgs.ankiAddons.puppy-reinforcement.withConfig {
        config = {
          encourage_every = 5;
        };
        userFiles = ./my-folder-of-the-most-cute-dogos;
      }
      ```
    '';
```